docs/howto/system_testing.md

```diff
@@ -26,6 +26,12 @@ Conceptually, running a system test involves the following steps:
 At the moment system tests have limitations. The salient ones are:
 * There isn't a way to do assert that the indexed data matches data from a file (e.g. golden file testing).
 
+## Stack version
+
+The tests use the [default version](https://github.com/elastic/elastic-package/blob/main/internal/install/stack_version.go#L9) `elastic-package` provides.
```

Comment:
What do you mean by that? Tests don't start the cluster, so can't have an influence on its version. In the Integrations repository, we select the stack based on Kibana version constraints.

If you're referring to the elastic-package repository, please move this point to the "Continuous Integration" section, as this file contains generic information also valid for Integrations repository.